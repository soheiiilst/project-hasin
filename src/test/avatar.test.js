import React from 'react';
import { shallow } from 'enzyme';
import Avatar from '../component/avatar/avatar.component';

it('expect to render Avatar component', () => {
  expect(shallow(<Avatar />)).toMatchSnapshot();
})
