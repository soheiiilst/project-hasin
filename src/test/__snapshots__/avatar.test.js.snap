// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Avatar component 1`] = `ShallowWrapper {}`;
